packages/wallets/src/components/DerivAppsSection/DerivAppsSection.scss

```diff
@@ -64,6 +64,22 @@
             height: 3.2rem;
         }
     }
+
+    &__button {
+        height: 3.2rem;
+        width: 3.2rem;
+        border-radius: 1.2rem;
+        background: none;
+        border: 1px solid var(--solid-slate-1400, #000);
```

Comment:
There's no css variable with this name `var(--solid-slate-1400, #000)`. Correct me if I'm wrong

Reply:
hey @aizad-deriv as i remember correctly we are taking this variables from figma, and then as a fallback we are adding hex color code.
i copied this var from figma and suggest to use it here

Reply:
@aizad-deriv is there a substitute css variable for #000?

Reply:
@heorhi-deriv I understand that but the naming of the variables in figma might not be the same as the one in codebase.  Although, there's is a fallback but if the variable doesn't exist in the css root then it will be better just to have the hexcode `#000`.

@lubega-deriv I suggest to use `var(--icon-black-plus, #000)` based on what we have alternatively you can refer to the `themes.scss` and `constants.scss` for the exact values that meet your needs.

Reply:
Updated thanks